src/python/doc/persistence_graphical_tools_user.rst

```diff
@@ -61,7 +60,7 @@ of shape (N x 2) encoding a persistence diagram (in a given dimension).
     import gudhi
     import numpy as np
     d = np.array([[0., 1.], [1., 2.], [1., np.inf]])
-    gudhi.plot_persistence_diagram(d)
+    gudhi.plot_persistence_diagram(d, legend=False)
```

Comment:
Maybe the default could be to show the legend when we pass a diagram of the form list[(dim,(birth,death))] but not when we pass a list[(birth, death)] where it is meaningless?

Reply:
Done on [9b5e21f](https://github.com/GUDHI/gudhi-devel/pull/734/commits/9b5e21f79bdf57ea63dc4b1aa7b08513e12fe54e). It avoids to specify the legend argument in the documentation.
We could also detect in `dimensions = set(item[0] for item in persistence)` the number of elements in `dimensions` to display or not the legend. To be discussed.